spec/get_player_input_spec.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/get_player_input.rb'
+require 'stringio'
+
+
+describe "#get_user_input" do
+    let(:board){Array.new(3){Array.new(3, "_")}}
+    it "should record player X's input in the board" do
+        input = StringIO.new("1,1\n")
+        get_player_input(input, "X", board)
+        expected = [['_', '_', '_'],['_', 'X', '_'], ['_', '_', '_']]
+        expect(board).to eq(expected)
+    end
+    it "should record player O's input in the board" do
+        input = StringIO.new("0,0\n")
+        get_player_input(input, "O", board)
+        expected =  [['O', '_', '_'],['_', '_', '_'], ['_', '_', '_']]
+        expect(board).to eq(expected)
+    end
+    it "should record second move for X" do
```

Comment:
My guess is that writing this test didn't force you to write any new code.  If that was the case, it's ok to write the test, see that it passes without changing the production code, and delete it.